spec/lib/miq_expression/field_spec.rb

```diff
@@ -70,6 +70,13 @@
       tag = "Vm.hosts.user_tag-name"
       expect(described_class.parse(tag)).to be_nil
     end
+
+    it 'parses field with numbers in column' do
```

Comment:
oh good :eyes I don't know why I am talking about column when  the bug is in association

(I am not aware about numbers in colums, at least for now :) )

thanks!!!
I am going to fix descriptions.